test/core/tsi/alts/fake_handshaker/fake_handshaker_server.cc

```diff
@@ -236,6 +236,7 @@ class FakeHandshakerService : public HandshakerService::Service {
     result.mutable_local_identity()->set_service_account("local_identity");
     string key(1024, '\0');
     result.set_key_data(key);
+    result.set_max_frame_size(16384);
```

Comment:
I am wondering how Envoy consume this value, especially we hardcode 16k here. What is Envoy change frame size to a different value?

Reply:
The change in this PR will be used only in the test, so it will not have any effect on envoy itself. By setting the frame size in the result, we want to verify [this code path](https://github.com/grpc/grpc/blob/master/src/core/tsi/alts/handshaker/alts_tsi_handshaker.cc#L172-L179) is indeed executed.

Reply:
SG